partner_delivery_zone/readme/CONFIGURATION.rst

```diff
@@ -1,3 +1,3 @@
-To configure this module you need to:
+To configure this module, you need to:
```

Comment:
The file should be named `CONFIGURE.rst` for being recognized.

Reply:
Changes made, and module just pushed.